### ISSUE TITLE
Add default support for timestamp types

### ISSUE THREAD

**Issue by gappleto97** [vsergeev/u-msgpack-python#34]
It would be nice if the new timestamp extension could be used without needing to provide an exthook for it. Suggested deserialization would be as the result of `datetime.datetime.utcfromtimestamp`

**Comment by vsergeev** [comment]
The new timestamp extension should be supported as of v2.5.0. Are you having issues packing/unpacking it?

**Comment by gappleto97** [comment]
Yes, actually. It doesn't seem to support the result of `datetime.utcnow()` on Python 3, even though you can on Python 2.

**Comment by gappleto97** [comment]
Python 2:

```python
>>> from datetime import datetime
>>> from umsgpack import packb
>>> packb(datetime.utcnow())
'\xd7\xff\xeb\xa5\xce@[-\xaeU'
```

Python 3:

```python
>>> from datetime import datetime
>>> from umsgpack import packb
>>> packb(datetime.utcnow())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/gappleto/.local/lib/python3.6/site-packages/umsgpack.py", line 607, in _packb3
    _pack3(obj, fp, **options)
  File "/home/gappleto/.local/lib/python3.6/site-packages/umsgpack.py", line 529, in _pack3
    _pack_ext_timestamp(obj, fp, options)
  File "/home/gappleto/.local/lib/python3.6/site-packages/umsgpack.py", line 347, in _pack_ext_timestamp
    delta = obj - _epoch
TypeError: can't subtract offset-naive and offset-aware datetimes
```

**Comment by vsergeev** [comment]
This issue was due to inconsistent handling of naive and aware datetime objects. I've fixed it in 64dfc38 to handle both types correctly.
